appserver/pom.xml

```diff
@@ -142,7 +142,7 @@
         <jms-api.version>2.0.1</jms-api.version>
 
         <!-- 5.1.1-b02.payara-p1 -->
-        <mq.version>5.1.1.final.payara-p4</mq.version>
+        <mq.version>5.1.2-SNAPSHOT</mq.version>
```

Comment:
This should be left at 5.1.1.final.payara-p4

Reply:
Sure. That seems to be branch bleeding. Sorry for this and thanks for the catch.